README.md

```diff
@@ -301,6 +302,14 @@ window/example:
+| -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `convertBytes` | Convert a given number (bytes) into a custom string format. <br><br>**Parameters**<br><br> - `bytes`: _`number`_ Bytes to convert.<br> - `decimals`: _`number`_ The number of decimals to convert the bytes up to. <br> - `minUnit`: _`string`_ The min unit to convert the bytes to. There are 3 options: <br>&nbsp;&nbsp;1, bits ('b', 'Kb', 'Mb', 'Gb', 'Tb', 'Pb', 'Eb', 'Zb', 'Yb')<br>&nbsp;&nbsp;2, bytes using SI standard ('B', 'KB', 'MB', ... , 'YB')<br>&nbsp;&nbsp;3, bytes using IEC standard (' B', 'KiB', 'MiB', ... , 'YiB').<br><br>**Examples:**<br><br> - `convertBytes(1024, 2, 'b')` -> `8.19 Kb`<br> - `convertBytes(1024, 2, 'B')` -> `1.02 KB`<br> - `convertBytes(1024, 2, ' B')` -> `1.00 KiB`<br> - `convertBytes(500, 1, ' B')` -> `500.0 B`<br> - `convertBytes(2000, 2, 'KiB')` -> `1.95 KiB` | `string`    | <img src="https://github.com/glzr-io/zebar/assets/34844898/568e90c8-cd32-49a5-a17f-ab233d41f1aa" alt="microsoft icon" width="24"><img src="https://github.com/glzr-io/zebar/assets/34844898/005a0760-da9d-460e-b533-9b2aba7f5c03" alt="apple icon" width="24"><img src="https://github.com/glzr-io/zebar/assets/34844898/1c5d91b1-879f-42a6-945e-912a11daebb4" alt="linux icon" width="24"> |
```

Comment:
Could we change the `convertBytes` fn to be under a general utility provider instead (e.g. `util`)?

Reply:
@lars-berger I renamed the data provider to util provider

Since we have util provider now, is there a reason for the date provider to be kept? It could simply be in the util provider

Reply:
The `data` provider? Nah we can remove it until there's a use case for it

Reply:
I was thinking about the `date` provider that has the toFormat function.

I initially added the `data` provider because there was no general utility provider (which we now do) and I always thought that functions like `toFormat` on the date provider should be separated so you could use it without having to use the date provider just to format a date from other providers. 
If for example you want to format a date field from like the `weather` provider then you use `util.formatDate` instead of `date.toFormat`. Not sure how you prefer, but now we use `util.convertBytes` instead of `data.convertBytes`. I thought that the same logic can apply for the `toFormat`